tests/phpunit/Persistence/ElasticValueCounterTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare( strict_types = 1 );
+
+namespace ProfessionalWiki\WikibaseFacetedSearch\Tests\Persistence;
+
+use PHPUnit\Framework\TestCase;
+use ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension;
+use Wikibase\DataModel\Entity\NumericPropertyId;
+
+/**
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\Persistence\ElasticValueCounter
+ * @covers \ProfessionalWiki\WikibaseFacetedSearch\WikibaseFacetedSearchExtension
+ */
+class ElasticValueCounterTest extends TestCase {
+
+	public function testCanExecuteValueCountQuery(): void {
+		$counter = WikibaseFacetedSearchExtension::getInstance()->getValueCounter();
+		$counter->countValues( new NumericPropertyId( 'P22' ) );
+		$this->assertTrue( true );
```

Comment:
I verified locally that I am getting the expected results.

Creating more meaningful tests runs into a test isolation challenge. AFAIK MediaWiki has no "test instance" of Elastic. Creating something ourselves that integrates with MW likely takes quite some effort, which I am hesitant to embark upon.